_docs/guides/integrating-vms.md

```diff
@@ -31,34 +31,41 @@ this infrastructure as a single mesh.
   [Installation guide]({{home}}/docs/setup/kubernetes/quick-start.html).
 
 * Deploy the [BookInfo]({{home}}/docs/guides/bookinfo.html) sample application.
-
-* Create a VM named 'db' in the same project as Istio cluster, and [Join the Mesh]({{home}}/docs/setup/kubernetes/mesh-expansion.html).
+* Create a VM named 'mysqldb' in the same project as Istio cluster, and [Join the Mesh]({{home}}/docs/setup/kubernetes/mesh-expansion.html).
 
 ## Running mysql on the VM
 
 We will first install mysql on the VM, and configure it as a backend for the ratings service.
 
 On the VM:
 ```bash
-  sudo apt-get update && apt-get install ...
-  # TODO copy or link the istio/istio test script
-```
+  sudo apt-get update && sudo apt-get install -y mariadb-server
+  sudo mysql
+  # Grant remote access by root to db server
+  GRANT ALL PRIVILEGES ON *.* TO 'root'@'%' IDENTIFIED BY 'password' WITH GRANT OPTION;
```

Comment:
with the sidecar this shouldn't be necessary ? (all connections appear to come from localhost) no ?